packages/yoga/src/Datepicker/web/Calendar.jsx

```diff
@@ -106,6 +106,7 @@ const DayField = styled.div`
     p {
         color: ${getDayFieldColor(selected, disabled, colors, aux)};
         z-index: 1;
+        position: absolute;
```

Comment:
This `position: absolute` is necessary?

Reply:
Yes, it is. Without the position absolute we get this:

![image](https://github.com/gympass/yoga/assets/18193620/ddcbceb1-2a3a-42e3-b721-964f0df19694)

Reply:
Actually, it's a problem when we are using the v3Theme for some reason we got this problem even in production:

https://gympass.github.io/yoga/components/datepicker

![image](https://github.com/gympass/yoga/assets/18193620/88a0c038-714a-484c-bad7-0dbfa4b27fc4)